packages/11ty/_plugins/markdown/index.js

```diff
@@ -22,8 +22,11 @@ const removeMarkdown = require('remove-markdown')
 module.exports = function(eleventyConfig, options) {
   /**
    * @see https://github.com/valeriangalliat/markdown-it-anchor#usage
+   * To prevent duplicate element IDs from slugified headings, we are only generating anchor links for level 1 headings
    */
-  const anchorOptions = {}
+  const anchorOptions = {
+    level: [1]
+  }
```

Comment:
@mphstudios this is a kinda dumb hack to remove duplicate IDs -- configuring automatic anchor linking to only occur on H1s -- might be better to just remove auto-anchor linking if that's what we really want to happen?

It would be nice to set this configuration per-output, but as far as I can tell that is not possible given the current way we handle different outputs individually in components. We could also strip out all H2 ids across the board in PDF output in the transform as well?

Reply:
@anderspollack I think this is good solution. Let's merge these changes.